doc/conftest.py

```diff
@@ -65,7 +65,7 @@ def pytest_runtest_setup(item):
         setup_rcv1()
     elif fname.endswith('datasets/twenty_newsgroups.rst'):
         setup_twenty_newsgroups()
-    elif fname.endswith('datasets/working_with_text_data.rst'):
+    elif fname.endswith('tutorial/text_analytics/working_with_text_data.rst'):
```

Comment:
Hmmm ... I am wondering why this actually worked before ...

Reply:
Oh well, I guess the 20 newsgroup data was just downloaded on Travis.

Reply:
Yep see https://travis-ci.org/scikit-learn/scikit-learn/jobs/305782018#L2491 where `doc/tutorial/text_analytics/working_with_text_data.rst` was run (`.` instead of `s`)

Reply:
And now it is skipped again:
https://travis-ci.org/scikit-learn/scikit-learn/jobs/305841207#L2491

Reply:
I was also wondering that! :-)